ozy/amr/read_amr_module.f90

```diff
@@ -481,6 +490,32 @@ subroutine select_from_descriptor_IDs(newVar,newID)
             varIDs%xHeII = newID
         case ('xHeIII')
             varIDs%xHeIII = newID
+        case ('chem_H')
```

Comment:
This is good, but it depends on the variables being properly written in the hydro_file_descriptor.